unittests.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><testsuite name="nosetests" tests="5" errors="0" failures="0" skip="0"><testcase classname="tests.test_inventory_items.TestInventoryItemModel" name="test_deserialize" time="0.028"></testcase><testcase classname="tests.test_inventory_items.TestInventoryItemModel" name="test_deserialize_bad_data" time="0.019"></testcase><testcase classname="tests.test_inventory_items.TestInventoryItemModel" name="test_deserialize_missing_data" time="0.016"></testcase><testcase classname="tests.test_inventory_items.TestInventoryItemModel" name="test_repr" time="0.016"></testcase><testcase classname="tests.test_inventory_items.TestInventoryItemModel" name="test_serialize" time="0.059"></testcase></testsuite>
```

Comment:
Same thing here, we don't want to commit `unittests.xml`. @Imsiaocong, can you please remove this file from the Pull Request?